Jenkinsfile

```diff
@@ -15,34 +15,36 @@ pipeline {
         }
         stage('bootstrap') {
             steps {
-                sh './bootstrap.sh'
+                sh '''
+                 carthage bootstrap --platform ios
+                '''
```

Comment:
There's no need to make this a multi-line string as it's still just a single command.